.github/workflows/ci.yml

```diff
@@ -648,6 +648,22 @@ jobs:
         run: ./build/linux/amd64/earthly config global.local_registry_host 'tcp://127.0.0.1:8371'
       - name: Build and push +prerelease
         run: ./build/linux/amd64/earthly --ci --push +prerelease
+      - name: Update DockerHub description for earthly/earthly
+        uses: peter-evans/dockerhub-description@v2
```

Comment:
Can we trust Peter?

Reply:
is there a way to pin it to a git sha to avoid it from changing?

Reply:
or alternative, we could fork this action to prevent it from changing.

Reply:
This action appears to be widely used; and what I've seen of the code when I was poking around it looked clean. I'll look to see if we can pin it? I don't think there is significant reason to worry though.

Reply:
https://julienrenaux.fr/2019/12/20/github-actions-security-risk/ makes some good points about why we should create a fork (to protect us from the repo being compromised or deleted).

Reply:
I would rather just script it than maintain a fork. Ill switch it to a hand-rolled script then? Or is the SHA pinning enough.

Do we have other actions we should be forking too that I could handle at the same time?

Reply:
pinning a sha1 is sufficient to prevent the issue of the repo being compromised.

Reply:
Latest commit should have it pinned.

Reply:
> Do we have other actions we should be forking too that I could handle at the same time?

Aside from the official github actions, we also use `docker/setup-qemu-action@v1` but it doesn't make use of credentials.